process-dataset/steps/upload-cell-counts.js

```diff
@@ -0,0 +1,32 @@
+const fsPromises = require('fs').promises;
+
+const map = require('lodash').map
+
+const {
+    TEMP_LOCAL_FILE_INFO_JSON
+} = require("../constants");
+
+const uploadCellCountsPerCellLine = async (readFolder, firebaseHandler) => {
+    console.log("uploading cell line counts... ")
+    const data = await fsPromises.readFile(`${readFolder}/${TEMP_LOCAL_FILE_INFO_JSON}`);
+    const json = JSON.parse(data);
+    const counts = json.reduce((acc, ele) => {
+
+        const cellLine = ele.CellLineName;
+        if (!acc[cellLine]) {
```

Comment:
I have no idea if this will actually matter in this context, but would it be safer to do `if (acc[cellLine] === undefined)` or `if (acc[cellLine] === null)`? Just made me pause because we're checking if `acc[cellLine]` is false-y and then setting it to 0, which would still be false-y

Reply:
It shouldn't matter because I'm setting it to 0 and then immediately incrementing it to 1, so it will never be 0 with we go back through the loop